### PR TITLE
issue-7 update normalizer tests to use cucumber with TopologyTester

### LINKED ISSUE #7: Update test suite in trade-normalizer to use cucumber
TopologyTester is designed around creating records from Map<String,String>, which lends itself extremely well to cucumber. Should set up cucumber in the trade normalizer as an example of how to use the TopologyTester's full capabilities

### PR DESCRIPTION
**Pull request by GalateaWade**
Updated existing tests to use cucumber so it is much easier to read and update.

### CODE REVIEW

File: trade-normalizer/src/test/resources/cucumber/securityNormalization.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      | secId      | secIsin |
+
+    Given receive the following trade records:
+      | isin    | qty |
+      | secIsin | 10  |
+
+    Then output the following trade records:
+      | securityId | qty |
+      | secId      | 10  |
+
```
Comment (gala-emile) [inline trade-normalizer/src/test/resources/cucumber/securityNormalization.feature:16]: Personally, I am a better fan of the #--------- we use to separate tests. I find it's much easier when you have LOADS of cucumber tests than 2 spaces. Just food for thought :)

**Comment by gala-emile** [comment]
Also there are conflicts

File: trade-normalizer/src/test/resources/cucumber/securityNormalization.feature (hunk 1 above)
Comment (GalateaWade) [reply trade-normalizer/src/test/resources/cucumber/securityNormalization.feature:16]: true, but I think it's unreasonable to have LOADS of cucumber tests in the same feature file. Should add separate feature files for different types of functionality being tested.

**Comment by GalateaWade** [comment]
fixed conflicts

**Comment by GalateaWade** [comment]
fixed conflicts

**Review by gala-emile — APPROVED** [review]

**Pull request merged**